rust/parser/typeql.pest

```diff
@@ -263,12 +263,12 @@ undefinable = { undefine_from | undefine_function | undefine_struct | label }
 undefine_from = { undefine_annotation_from_capability
                 | undefine_annotation_from_type
                 | undefine_capability
-                | undefine_override
+                | undefine_specialise
                 }
 undefine_annotation_from_capability = { annotation_category ~ FROM ~ label ~ type_capability_base }
 undefine_annotation_from_type = { annotation_category ~ FROM ~ label }
 undefine_capability = { type_capability_base ~ FROM ~ label }
-undefine_override = { AS ~ label ~ FROM ~ label ~ type_capability_base }
+undefine_specialise = { AS ~ label ~ FROM ~ label ~ type_capability_base }
```

Comment:
Ideally, there should be only `relates`, but, as I understand, we prefer to use `capability` in general in these situations (parsing for pure `relates` declarations is private, and it previously combined all the capabilities instead of just 3: `owns`, `relates`, `plays`). So I'm going to leave it here and just add errors on the server side ("You tried to undefine specialisation for `owns`/`sub`/..., but this feature doesn't exist for this capability).

Reply:
Use `pub(in crate::parser)`

Reply:
Done